etc/profile-a-l/0ad.profile

```diff
@@ -15,6 +15,23 @@ blacklist /usr/libexec
 include disable-common.inc
 include disable-devel.inc
 include disable-exec.inc
+#
+# Start fix for OpenSUSE Tumbleweed
+# =================================
+#
+# Allow mozjs (blacklisted by disable-interpreters.inc)
+# This code  has to be placed before:
+# include disable-interpreters.inc
+#
+include allow-gjs.inc
+#
+# which contains these lines:
+# noblacklist /usr/lib/libmozjs-*
+# noblacklist /usr/lib64/libmozjs-*
+#
+# ===============================
+# End fix for OpenSUSE Tumbleweed
+#
```

Comment:
Just

```suggestion
# Allow gjs (blacklisted by disable-interpreters.inc)
include allow-gjs.inc
```

should be enough. And move it up in a new block between `noblacklist ${HOME}/.local/share/0ad` and `blacklist /usr/libexec`.


If you do not know how you can edit it again: go to your fork, switch the branch to the one of this PR and navigate to 0ad.profile. Check that you are at `https://github.com/leukimi/firejail/blob/master-1/etc/profile-a-l/0ad.profile`, edit and commit directly.